ui/job-view/pushes/JobGroup.jsx

```diff
@@ -137,6 +151,16 @@ export class JobGroupComponent extends React.Component {
             tier={groupTier}
             toggleExpanded={this.toggleExpanded}
           />
+          {runnableVisible && expanded && (
+            <button
+              className="btn group-select-all-runnable"
+              type="button"
+              title="Select or deselect all jobs in this group"
+              onClick={this.toggleAll.bind(this)}
```

Comment:
For future reference, you shouldn't need to bind to `this` if the function you are calling is using the fat arrow implementation (you can see this is done elsewhere in the codebase).